packages/itwin/tree-widget/src/components/trees/imodel-content-tree/IModelContentTree.tsx

```diff
@@ -21,24 +20,15 @@ export const RULESET_IMODEL_CONTENT: Ruleset = require("./IModelContent.json");
  * Props for [[IModelContentTree]]
  * @public
  */
-export interface IModelContentTreeProps extends Omit<React.HTMLProps<HTMLDivElement>, "children"> {
-  /** An IModel to pull data from */
-  iModel: IModelConnection;
-  /** Width of the component */
-  width: number;
-  /** Height of the component */
-  height: number;
-  /** Selection mode in the tree */
-  selectionMode?: SelectionMode;
-}
+export type IModelContentTreeProps = BaseTreeProps;
```

Comment:
isnt this technically breaking since the type is getting narrowed?
should be ok since this is a pre-release atm

Reply:
Yes. I think I missed this one when working on 1.0 so it's good that it was caught while it's still pre-release version. I can add a separate change file for this.

Reply:
Added separate change message to highlight that IModelContentTreeProps were broken.